package.json

```diff
@@ -5,6 +5,10 @@
   "version": "0.1.0",
   "author": "Kyle Mathews <mathews.kyle@gmail.com>",
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.21",
```

Comment:
We already have React-Icons, which includes font awesome already

Reply:
Didn't notice, already ported to 'react-icons'.